packages/react/src/components/stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,307 @@
+import React, { useReducer } from 'react';
+
+import { Stepper } from './Stepper';
+import styles from './Stepper.module.scss';
+import { Step } from './Step';
+import { Button } from '../button';
+
+export default {
+  component: Stepper,
+  title: 'Components/Stepper',
+  decorators: [(storyFn) => <div style={{ maxWidth: '700px' }}>{storyFn()}</div>],
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+const commonReducer = (stepsTotal) => (state, action) => {
+  switch (action.type) {
+    case 'completeStep': {
+      const activeStep = action.payload === stepsTotal ? stepsTotal : action.payload + 1;
+      return {
+        activeStep,
+        states: state.states.map((stateName, index) => {
+          if (index === action.payload - 1 && index !== stepsTotal - 1) {
+            // current one but not last one
+            return 'filled';
+          }
+          if (index === action.payload) {
+            // next one
+            return 'available';
+          }
+          return stateName;
+        }),
+      };
+    }
+    case 'setActive': {
+      return {
+        activeStep: action.payload,
+        states: state.states.map((stateName, index) => {
+          if (index === action.payload - 1) {
+            return 'available';
+          }
+          return stateName;
+        }),
+      };
+    }
+    default:
+      throw new Error();
+  }
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Default = (args) => {
+  const reducer = commonReducer(5);
+
+  const initialState = {
+    activeStep: 1,
+    states: ['available', 'disabled', 'disabled', 'disabled', 'disabled'],
+  };
+
+  const labels = ['Step 1', 'Step 2', 'Step 3', 'Step 4 - longer text', 'Step 5'];
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  return (
+    <div>
+      <Stepper
+        labels={labels}
+        language="en"
+        states={state.states}
+        selectedStep={state.activeStep}
+        stepsTotal={5}
+        onStepClick={(event, number) => dispatch({ type: 'setActive', payload: number })}
+      />
+
+      <div style={{ height: '300px', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+        <Button
+          onClick={() => dispatch({ type: 'completeStep', payload: state.activeStep })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+        >
+          Complete step {state.activeStep}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Small = (args) => {
```

Comment:
On mobile screen sizes, I would expect the Stepper to be full-width.

![mobile-small](https://user-images.githubusercontent.com/4214671/151328055-0905262d-e398-4003-bf3d-10857f5dc345.jpg)

Reply:
The width of the stepper is configured outside of the component. Here it was set to be some specific width for a demo purposes. We could change the width to be full-width, but since we are trying to demo the small variant here, I thought it would be better to show how it scales on smaller width.

Reply:
- [ ] Make stepper full width for small story